plugins/version.py

```diff
@@ -14,7 +15,11 @@ def version(bot, mask, target, args):
     """
 
     try:
-        repo = Repo()
+        """Get path of plugins folder and derive git root"""
+        git_root = os.path.dirname(os.path.realpath(__file__))[:-8]
```

Comment:
os.path.dirname(os.path.realpath(__file__)) returns the plugins directory, [:-8] removes "plugins/" because this sub-directoryname is static.